upgrade/upgrade-scripts/upgrade-container

```diff
@@ -262,6 +262,27 @@ function create_upgrade_container() {
 		rpool/crashdump           /var/crash    zfs defaults,x-systemd.before=zfs-import-cache.service,x-systemd.before=kdump-tools.service 0 0
 	EOF
 
+	#
+	# DLPX-75089 - Since older versions of Delphix did not properly
+	# disable the NFS services within the upgrade container, we have
+	# to dynamically disable them here. This way, when the container
+	# is started, those NFS services will not run (no matter the
+	# version of Delphix we're upgrading from).
+	#
+	for svc in "nfs-mountd" "nfs-server" "rpc-statd"; do
```

Comment:
to be complete, we should also include `rpc-statd-notify` service.  It only runs on a reboot but I'm not sure if starting the container acts like a reboot.

Reply:
Thanks, done.